src/renderer/scss/component/_modal.scss

```diff
@@ -18,11 +18,13 @@
 }
 
 .modal {
+  /* Set modal buttons background color */
+  --btn-bg-alt: var(--modal-btn-bg-color);
```

Comment:
Do we need this? It doesn't look like it's doing anything. We should set any variables in `vars.scss`